### DISCUSSION TITLE
How to swap one key and its shifted-variant?  `9 <-> (` and `0 <-> )`

### DISCUSSION THREAD

**Discussion by jian-lin** [kmonad/kmonad#939] [Q&A] [answered]
I failed to find a config for this after reading the quick reference.

**Comment by jokesper** [comment]
Specify `9` and `0` in your `defsrc` and specify `(` and `)` in `deflayer`

**Reply by jian-lin** [reply]
This only gets half what I want.  It failed to trun `(` into `9` and `)` into `0`.

**Reply by jokesper** [reply, marked as answer]
This is a bit tricky.
you could make shift a layer-toggle and have them there. To get the remaining keys shifted, you could either specify them all or use `(around sft (layer-toggle ...))` with this you would have to unpress shift. via something like `#((release-only sft) 9 (press-only`sft))`. For this problem there exists already an issue: #111
